server/utils/placements/index.ts

```diff
@@ -169,6 +172,32 @@ export const departureInformation = (placement: Cas1SpaceBooking): SummaryList =
   }
 }
 
+export const requirementsInformation = (placement: Cas1SpaceBooking): SummaryList => {
+  const requirements = placement.requirements.essentialCharacteristics
+  const apType =
+    apTypeCriteriaLabels[requirements.find(requirement => specialistApTypeCriteria.includes(requirement)) || 'normal']
+  const apRequirements = filterApLevelCriteria(requirements)
+  const roomRequirements = filterRoomLevelCriteria(requirements)
+
+  const noneSelected = `<span class="text-grey">None</span>`
```

Comment:
Nice.
There are probably other places where this could be used. In fact, couldn't `requirementsHtmlString` return it if the array is empty?

Reply:
I was just thinking about that as I'd started looking at the actual issue with 'change placement' when criteria is empty. I'll merge this and apply it!